layouts/shortcodes/dashboard-table.html

```diff
@@ -0,0 +1,107 @@
+                <!-- Weight -->
+                <td>{{- (default ($scratch.Get "dashboard-weigth") .Params.dashboardWeight) -}}</td>
+                <!-- State -->
+                {{ $color := "#6c8293"}}
+                {{- if eq .Params.dashboardState "incorrect" -}}
+                    {{ $color = "#ff0010" }}
+                {{- else if eq .Params.dashboardState "wip" -}}
+                    {{ $color = "#ff6f00" }}
+                {{- else if eq .Params.dashboardState "incomplete" -}}
+                    {{ $color = "#ffef00" }}
+                {{- else if eq .Params.dashboardState "stable" -}}
+                    {{ $color = "#90ff00" }}
+                {{- end -}}
+                <td style="background-color:{{- $color -}};color: black">{{- (default "N/A" (humanize .Params.dashboardState)) -}}</td>
+                <!-- Audit -->
+                {{- $audit := (default ($scratch.Get "dashboard-audit") .Params.dashboardaudit) -}}
+                {{ $color1 := "#6c8293"}}
+                {{- if eq $audit 0 -}}
+                    {{ $color1 = "#ff0010" }}
+                {{- else if gt $audit 0 -}}
+                    {{ $color1 = "#90ff00" }}
+                {{- end -}}
+                <td style="background-color:{{- $color1 -}};color: transparent;">{{$audit}}</td>
+                <!-- Tests -->
+                {{- $tests := .Params.dashboardTests -}}
+                {{ $color2 := "#6c8293"}}
+                {{- if eq $tests 0 -}}
+                    {{ $color2 = "#ff0010" }}
+                {{- else if gt $tests 0 -}}
+                    {{ $color2 = "#90ff00" }}
+                {{- end -}}
+                <td style="background-color:{{- $color2 -}};color: black"></td>
+            </tr>
+            {{ template "dashboard-section-children" (dict "Section" . "level" $level) }}
+        {{ else if and .IsPage .Content }}
+        <tr>
+            <td style="max-width:250px;white-space: nowrap;overflow: hidden;text-overflow: ellipsis;">{{- delimit (split $level "") "." }} <a title="{{.Title}}" href="{{.Permalink}}">{{.Title}}</a></td>
+            <!-- Weight -->
+            <td>{{- (default 0 .Params.dashboardWeight) -}}</td>
+            <!-- State -->
+            {{ $color := "#6c8293"}}
+            {{- if eq .Params.dashboardState "incorrect" -}}
+                {{ $color = "#ff0010" }}
+            {{- else if eq .Params.dashboardState "wip" -}}
+                {{ $color = "#ff6f00" }}
+            {{- else if eq .Params.dashboardState "incomplete" -}}
+                {{ $color = "#ffef00" }}
+            {{- else if eq .Params.dashboardState "stable" -}}
+                {{ $color = "#90ff00" }}
+            {{- end -}}
+            <td style="background-color:{{- $color -}};color: black">{{- (default "N/A" (humanize .Params.dashboardState)) -}}</td>
```

Comment:
How about pulling the colors into css classes for each state, so we can drop the if/else color blocks
```css
.bg-incorrect { background-color: #ff0010 }
.bg-wip { background-color: #ff6f00 }
.bg-incomplete { background-color: #ffef00 }
.bg-stable { background-color: #90ff00 }
.black { color: #111 } 
```
```suggestion
            <td class="{{- .Params.dashboardState -}} black">{{- (default "N/A" (humanize .Params.dashboardState)) -}}</td>
```